codelists/migrations/0050_set_codelistversion_author.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 4.0.3 on 2022-03-17 11:16
+
+from django.db import migrations
+
+
+def set_author_from_draft_owner(apps, schema_editor):
+    CodelistVersion = apps.get_model("codelists", "CodelistVersion")
+    versions_with_draft_owner = CodelistVersion.objects.filter(
+        draft_owner__isnull=False
+    )
+    for version in versions_with_draft_owner:
+        version.author = version.draft_owner
+        version.save()
```

Comment:
Is it worth using an `.update()` call here?

Reply:
It's not slow, but it could

Reply:
Yeah I imagine we don't have anywhere near enough versions to see perf gains here!

Reply:
I added it anyway, just because :slightly_smiling_face: